packages/jupyter-ai/jupyter_ai/extension.py

```diff
@@ -106,6 +115,20 @@ class AiExtension(ExtensionApp):
         config=True,
     )
 
+    default_model_provider_id = Unicode(
+        default_value=None,
+        allow_none=True,
+        help="Default language model provider.",
+        config=True,
+    )
+
+    default_embeddings_provider_id = Unicode(
+        default_value=None,
+        allow_none=True,
+        help="Default embeddings model provider.",
+        config=True,
+    )
```

Comment:
As noted in the docs, these parameters might be misnamed; they are stated to be *provider IDs*, but these are really *model IDs*.

```suggestion
    default_language_model = Unicode(
        default_value=None,
        allow_none=True,
        help="Default language model.",
        config=True,
    )

    default_embedding_model = Unicode(
        default_value=None,
        allow_none=True,
        help="Default embedding model.",
        config=True,
    )
```